examples/monaco-graphql-webpack/babel.config.js

```diff
@@ -1,14 +1 @@
-module.exports = {
```

Comment:
We are using the root babel config for the `graphiql` package. This removal should be safe.
Plus it fixes the lint error.